internal/pkg/template/env.go

```diff
@@ -215,6 +215,7 @@ type SecurityGroupRule struct {
 	ToPort     int
 }
 
+// VPCFlowLogs holds the fields to configure a FlowLog
```

Comment:
nit: https://github.com/golang/go/wiki/CodeReviewComments#comment-sentences 
> Comments should begin with the name of the thing being described and end in a period
```suggestion
// VPCFlowLogs holds the fields to configure logging IP traffic using VPC flow logs.
```
😅